projects/packages/jitm/src/js/jetpack-jitm.js

```diff
@@ -300,23 +301,18 @@ jQuery( document ).ready( function ( $ ) {
 
 			var full_jp_logo_exists = $( '.jetpack-logo__masthead' ).length ? true : false;
 
-			$.get( window.jitm_config.api_root + 'jetpack/v4/jitm', {
-				message_path: message_path,
-				query: query,
-				full_jp_logo_exists: full_jp_logo_exists,
-				_wpnonce: $el.data( 'nonce' ),
-			} ).then( function ( response ) {
-				if ( 'object' === typeof response && response[ '1' ] ) {
-					response = [ response[ '1' ] ];
+			apiFetch( {
+				path: addQueryArgs( 'jetpack/v4/jitm', {
+					message_path: message_path,
+					query: query,
+					full_jp_logo_exists: full_jp_logo_exists,
+				} ),
+				method: 'GET',
+			} ).then( function ( messages ) {
+				const message = messages?.[ 0 ];
+				if ( message?.content ) {
+					setJITMContent( $el, message, redirect );
 				}
```

Comment:
This is addressing https://github.com/Automattic/jetpack/pull/41252/files#r1954353493